src/config/post-install.md

```diff
@@ -6,15 +6,8 @@ machine.
 ## Updates
 
 Installation media contains a snapshot of packages from the day it was made.
-After installing and connecting the network, new systems should be updated:
-
-```
-# xbps-install -Su
-```
+After installing and connecting the network, new systems should be
+[updated](../xbps/updating.md).
 
 This will ensure that your system has applicable security patches and software
 upgrades that happened after the installation media was created.
-
-> Note: XBPS must use a separate transaction to update itself. If your first
-> update includes the package `xbps`, you will need to run an additional update
-> for the rest of the system.
```

Comment:
Hmm, not sure about removing this. i'd prefer to keep it (unless this information is mentioned elsewhere), so that people don't end up thinking that the independent `xbps` package update was the only update necessary.

Reply:
I removed this one because it's exactly the same one that is in the upgrading page.

Reply:
Ah okay, fair enough.